app/controllers/practices_controller.rb

```diff
@@ -10,8 +10,12 @@ class PracticesController < ApplicationController
                                            :documentation, :resources, :complexity,
                                            :timeline, :risk_and_mitigation,
                                            :contact, :checklist, :published,
-                                           :publication_validation
-  ]
+                                           :publication_validation]
+  before_action only: [:update] do
+    if date_initiated_params_exist(params[:date_initiated])
+      params[:practice][:date_initiated] = create_date_initiated(params[:date_initiated])
+    end
+  end
```

Comment:
this is interesting.
:thinking: 
i've never seen a `before_action` with a `do` before. 
usually there is a method and then the list of actions that the method pertains to.
like:
```
before_action :can_create_practice, only: [:new, :create]
```
i'll have to think about how i feel about this.
if i don't like it later we can change it :wink:  
i'll probably not like it if there are several actions in the list like on line 8

Reply:
yes! that sounds good to me! thank you for the feedback!